app/controllers/songs_controller.rb

```diff
@@ -12,6 +12,12 @@ def create
     redirect_to setlist_path(id: params[:setlist_id])
   end
 
+  def destroy
+    Song.destroy(params[:id])
+    SetlistSong.destroy(params[:setlist_song_id])
```

Comment:
do we ever actually want to destroy a song?  Wouldn't we just be looking to destroy a setlist song?  also don't we need to delete the associated song on spotify?